src/account/account.ts

```diff
@@ -59,3 +81,30 @@ export async function createUser(
 
   return { ...account, reference }
 }
+
+/**
+ * Creates a new user (version 2) and uploads the encrypted account to the network
+ *
+ * @param connection connection information for data uploading
+ * @param username FDP username
+ * @param password FDP password
+ * @param mnemonic mnemonic phrase
+ */
+export async function createUser(
+  connection: Connection,
+  username: string,
+  password: string,
+  mnemonic: string,
+): Promise<Reference> {
+  const { wallet, encryptedMnemonic } = await createUserAccount(password, mnemonic)
+  const topic = generateRandomBase64String(RANDOM_TOPIC_LENGTH)
+  const encryptedMnemonicAddress = removeZeroFromHex(
+    await writeFeedData(connection, topic, stringToBytes(encryptedMnemonic), wallet.privateKey),
+  )
+  const paddingLength = Math.floor(Math.random() * (PADDING_MAX - PADDING_MIN)) + PADDING_MIN
```

Comment:
Yes, most likely we need to do padding. Then the fill method must be subject to requirements regarding the use of characters (instead of base64, we need to use a random hex string). Because if the character sets are different, then you can also make an assumption about the hidden address. Is not it?

Reply:
so the padded characters should be completely random characters without any characteristic for the whole chunk payload.